src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ReactDOM from "react-dom";
```

Comment:
a bunch of these imports are unused too i believe

Reply:
yes, i will delete them now